test/e2eTests/tests/checkIn_test.js

```diff
@@ -41,6 +47,12 @@ After(async (I, Apis, Data) => {
     }
   };
 
+  if(!Patient1)
```

Comment:
Lint may complain about there not being a space after the if

Reply:
I have eslint running as part of my vs code setup, and I'm not seeing any messages. I'll run it from the CLI and see if there are any there.